google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadConversionAdjustment.java

```diff
@@ -0,0 +1,213 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.remarketing;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v3.enums.ConversionAdjustmentTypeEnum.ConversionAdjustmentType;
+import com.google.ads.googleads.v3.errors.GoogleAdsError;
+import com.google.ads.googleads.v3.errors.GoogleAdsException;
+import com.google.ads.googleads.v3.services.ConversionAdjustment;
+import com.google.ads.googleads.v3.services.ConversionAdjustmentResult;
+import com.google.ads.googleads.v3.services.ConversionAdjustmentUploadServiceClient;
+import com.google.ads.googleads.v3.services.GclidDateTimePair;
+import com.google.ads.googleads.v3.services.RestatementValue;
+import com.google.ads.googleads.v3.services.UploadConversionAdjustmentsResponse;
+import com.google.ads.googleads.v3.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.DoubleValue;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import javax.annotation.Nullable;
+
+/**
+ * Imports conversion adjustments for conversions that already exist. To set up a conversion action,
+ * run the AddConversionAction.java example.
+ */
+public class UploadConversionAdjustment {
+  private static class UploadConversionAdjustmentParams extends CodeSampleParams {
+
+    @Parameter(names = ArgumentNames.CUSTOMER_ID, required = true)
+    private long customerId;
+
+    @Parameter(names = ArgumentNames.CONVERSION_ACTION_ID, required = true)
+    private long conversionActionId;
+
+    @Parameter(names = ArgumentNames.GCLID, required = true)
+    private String gclid;
+
+    @Parameter(names = ArgumentNames.ADJUSTMENT_TYPE, required = true)
+    private String adjustmentType;
+
+    @Parameter(
+        names = ArgumentNames.CONVERSION_TIME,
```

Comment:
Better to rename the parameters to "CONVERSION_DATE_TIME",  "ADJUSTMENT_DATE_TIME" 

to be consistent with other libraries e.g. 
PHP https://github.com/googleads/google-ads-php/blob/master/examples/Remarketing/UploadConversionAdjustment.php#L51

Perl: https://github.com/googleads/google-ads-perl/blob/master/examples/remarketing/upload_conversion_adjustment.pl#L55